services/autoscaling/src/simcore_service_autoscaling/utils/rabbitmq.py

```diff
@@ -40,19 +40,31 @@ async def create_autoscaling_status_message(
 ) -> RabbitAutoscalingStatusMessage:
     assert app_settings.AUTOSCALING_EC2_INSTANCES  # nosec
     assert app_settings.AUTOSCALING_NODES_MONITORING  # nosec
-    total_resources, used_resources, all_ec2_instances = await asyncio.gather(
+    (
+        total_resources,
+        used_resources,
+        pending_ec2_instances,
+        running_ec2_instances,
+    ) = await asyncio.gather(
         *(
             utils_docker.compute_cluster_total_resources(monitored_nodes),
             utils_docker.compute_cluster_used_resources(docker_client, monitored_nodes),
-            ec2_client.get_all_pending_running_instances(
+            ec2_client.get_instances(
                 app_settings.AUTOSCALING_EC2_INSTANCES,
                 list(
                     ec2.get_ec2_tags(app_settings.AUTOSCALING_NODES_MONITORING).keys()
                 ),
+                state_names=["pending"],
+            ),
+            ec2_client.get_instances(
+                app_settings.AUTOSCALING_EC2_INSTANCES,
+                list(
+                    ec2.get_ec2_tags(app_settings.AUTOSCALING_NODES_MONITORING).keys()
+                ),
+                state_names=["running"],
```

Comment:
MINOR: it would be good to have a place that holds all the valid EC2 instance names. Ideally with a link to doc. 
Eventually we can create either ``Literal`` or ``(str, Enum)``  from them:

```python
# we could add this in ``servicelib``
class AutoEnumStr(str, Enum):
    def _generate_next_value_(name, start, count, last_values):
        return name

class EC2InstanceStates(AutoEnumStr):
    pending = auto()
    running = auto()
    ...
```
SEE https://docs.python.org/3.9/library/enum.html#using-automatic-values

```

Reply:
so there is a list of Literals. so typing will check it auto. not as strong as an enum but I find the enum a bit of an overkill for now.